docs/build/environment/local-network.md

```diff
@@ -56,7 +56,7 @@ You can see the full list of the command options using the `help` subcommand:
 # ...
 ```
 
-When you have successfully lauched the local network, you will see the following messages in your terminal:
+When you have successfully launched the local network, you will see the following messages in your terminal:
 
 ![1](img/1.png)
```

Comment:
Used image here is wrong.

Reply:
same for these two
- https://github.com/AstarNetwork/astar-docs/pull/253/files#diff-bd83349a8bd7602738ab0209bbec7d4bface067530563ac270a8961bef63c776R73
- https://github.com/AstarNetwork/astar-docs/pull/253/files#diff-bd83349a8bd7602738ab0209bbec7d4bface067530563ac270a8961bef63c776R77

Reply:
good catch!!! Thanks. Please check